src/App.css

```diff
@@ -1,3 +1,138 @@
-body {
-  background-color: midnightblue !important;
+h2 {
+  color: white !important;
+  margin-bottom: 0 !important;
+}
+
+.detail h4 {
+  color: #fff;
+}
+
+.details {
+  margin-top: 27px;
+}
+
+.card-item {
+  background-color: white;
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  gap: 10px;
+  margin-top: 30px;
+}
+
+div.grid a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.active-data {
+  background-color: #fd5294;
+  padding: 10px;
+}
+
+.inactive-data {
+  background-color: #ce4277;
+  padding: 10px;
+}
+
+h3 {
+  color: #fff !important;
+}
+
+.top {
+  display: flex;
+  align-items: baseline;
+  justify-content: space-between;
+}
+
+div.container h2 {
+  color: rgb(39, 39, 39) !important;
+  padding-top: 10px;
+}
+
+.detail {
+  display: flex;
+  justify-content: space-between;
+  padding: 20px;
+}
+
+.country {
+  background-color: #fd5294;
+}
+
+.total {
+  background-color: #ce4277;
+}
+
+.recover {
+  background-color: #ce4277;
+}
+
+.deaths {
+  background-color: #fd5294;
+}
+
+a img {
+  width: 27px;
+}
+
+section.top-page {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+button.btn {
+  margin-right: 11px;
+}
+
+a.active img {
+  display: none;
+}
+
+.bg-back {
+  background-color: #fd5294;
+}
+
+@media screen and (max-width: 480px) {
+  .grid {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 0;
+    margin-top: 0;
+  }
+
+  div.container {
+    padding: 0;
+  }
+
+  div.container h2 {
+    padding: 10px;
+    background-color: #fd5294;
+  }
+
+  div .container h2 {
+    color: rgb(255, 255, 255) !important;
+  }
+
+  .details {
+    margin-top: 0;
+  }
+
+  section.top-page {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .top-btns {
+    padding: 4px;
+    padding-left: 9px;
+    text-align: center;
+  }
+
+  button.btn.btn-outline-success {
+    margin-bottom: 5px;
+  }
 }
```

Comment:
Please ensure your design looks more like the original design from Behance, for a better UI.

Your Design  |  Behance Design
![Screenshot from 2023-08-12 19-06-19](https://github.com/ctokoli/react-capstone/assets/44624138/ea25ceca-d593-4d86-b565-488b4f935941)  |  ![Screenshot from 2023-08-12 19-08-23](https://github.com/ctokoli/react-capstone/assets/44624138/29c188f6-fff2-487e-a3cb-e03f9b1eee23)